docs/features/search/architecture.md

```diff
@@ -6,9 +6,8 @@ description: Documentation on Search Architecture
 
 # Search Architecture
 
-> _This is a proposed architecture which has not been implemented yet. We are
-> still looking for feedback to improve the architecture to fit your use-case,
-> see [this open issue](https://github.com/backstage/backstage/issues/4078)._
+> _This is a proposed architecture which has not been implemented yet. Find our
+> milestones to follow our progress [here](./README.md)._
```

Comment:
```suggestion
> milestones to follow our progress on the [Search Roadmap](./README.md#project-roadmap)._
```

A pet peeve of mine, but would it be possible to reword this?  (the dreaded avoiding "click here" 😁 
 https://www.nngroup.com/articles/top-ten-web-design-mistakes-of-2005/ )

I think this hash link will work...